README.md

```diff
@@ -49,6 +49,11 @@ Builds are provided for the following platforms:
 1. [Install Gstreamer][gstreamer] from the most recent MSI installer on Windows,
 or your package manager on Linux.
 
+    Ubuntu/Debian:
+        ```apt-get install libgstreamer1.0-dev libgstreamer-plugins-base1.0-dev libgstreamer-plugins-bad1.0-dev gstreamer1.0-plugins-base gstreamer1.0-plugins-good gstreamer1.0-plugins-bad gstreamer1.0-plugins-ugly gstreamer1.0-libav gstreamer1.0-doc gstreamer1.0-tools gstreamer1.0-x gstreamer1.0-alsa gstreamer1.0-gl gstreamer1.0-gtk3 gstreamer1.0-qt5 gstreamer1.0-pulseaudio
+        ```
```

Comment:
We don't require most of these. I think it best to keep the list to the minimal 

```suggestion
        ```bash
        sudo apt install \
          libgstrtspserver-1.0-0 \
          libgstreamer1.0-0 \
          libgstreamer-plugins-bad1.0-0 \
          gstreamer1.0-plugins-good \
          gstreamer1.0-plugins-bad
        ```
```

With the bash tag and the new lines to improve readability